pkg/fission-cli/cmd/client.go

```diff
@@ -84,17 +87,86 @@ func getLoadingRules() (loadingRules *clientcmd.ClientConfigLoadingRules, err er
 	return loadingRules, nil
 }
 
+// InClusterConfig returns a config object which uses the service account
+// kubernetes gives to pods. It's intended for clients that expect to be
+// running inside a pod running on kubernetes. It will return ErrNotInCluster
+// if called from a process not running in a kubernetes environment.
+func InClusterConfig() (*rest.Config, error) {
```

Comment:
As https://github.com/fission/fission/pull/2996/files uses "k8s.io/client-go/rest" API directy. We just prefer reusing k8s go code as much possibe.

Reply:
Thank you very much for the feedback, but above all, for implementing this very useful feature.